src/encoded/tests/data/master-inserts/static_section.json

```diff
@@ -710,7 +710,16 @@
         }
     },
 
-
+    {
+        "name": "experiment-set-data-plans.matrix",
+        "uuid": "d6bf6e5c-e0b4-41c4-bfb5-f56edc480f67",        
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5723113)